examples/workflows/misc/http.py

```diff
@@ -1,14 +1,26 @@
-from hera.expr import g
-from hera.workflows import HTTP, Parameter, Workflow
-
-with Workflow(generate_name="http-", entrypoint="http") as w:
-    HTTP(
-        name="http",
-        inputs=[Parameter(name="url", value="https://example.com")],
-        timeout_seconds=20,
-        url=f"{g.inputs.parameters.url:$}",
-        method="GET",
-        headers=[{"name": "x-header-name", "value": "test-value"}],
-        success_condition=str(g.response.body.contains("google")),  # type: ignore
-        body="test body",
-    )
+from hera.workflows import Parameter, Steps, Workflow, script
+from hera.workflows.models import ValueFrom
+
+
+@script(
+    outputs=[
+        Parameter(name="hello-output", value_from=ValueFrom(path="/tmp/hello_world.txt")),
+    ]
+)
+def hello_to_file():
+    with open("/tmp/hello_world.txt", "w") as f:
+        f.write("Hello World!")
+
+
+@script()
+def repeat_back(message: str):
+    print(f"You just said: '{message}'")
+
+
+with Workflow(
+    generate_name="hello-world-parameter-passing-",
+    entrypoint="steps",
+) as w:
+    with Steps(name="steps"):
+        hello_world_step = hello_to_file()
+        repeat_back(arguments={"message": hello_world_step.get_parameter("hello-output")})
```

Comment:
This example doesn't seem to be doing HTTP any more?

Reply:
Lol I was using it to generate YAML for the walkthrough examples and forgot to reset it. Thanks for spotting!